### PR TITLE
feat: Support "master" and "main" as the default branch

### PR DESCRIPTION
**Pull request by skovhus**
We now support both `master` and `main` as the default branch.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ jobs:
       - name: Get Deployment URL
         id: deployment-url
         run: |
-          SUBDOMAIN=$([[ $GITHUB_REF = "refs/heads/master" ]] && echo "" || echo "preview-${{ inputs.tree_hash }}.")
+          SUBDOMAIN=$([[ $GITHUB_REF = "refs/heads/master" || $GITHUB_REF = "refs/heads/main" ]] && echo "" || echo "preview-${{ inputs.tree_hash }}.")
```
Comment (pekala) [inline .github/workflows/deploy.yml]: Could potentially get the default branch as a workflow argument (defaulting to `main`)

**Review by pekala — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (skovhus) [reply .github/workflows/deploy.yml]: Yeah. I'll leave that as an optimization for now and add a comment.

Comment (skovhus) [reply .github/workflows/deploy.yml]: Actually, making this mandatory makes sense to me. https://github.com/pleo-io/pleo-spa-cicd/pull/27/commits/a2c56ad9bb4a3da7fd0b8626ba1844d51d1f2a0b

Comment (skovhus) [reply .github/workflows/deploy.yml]: Correction: we get this from the GitHub event already: `${{ github.event.repository.default_branch }}`

**Pull request merged**